src/panels/config/helpers/forms/ha-input_select-form.ts

```diff
@@ -16,6 +17,8 @@ import { HomeAssistant } from "../../../../types";
 import { fireEvent } from "../../../../common/dom/fire_event";
 import { haStyle } from "../../../../resources/styles";
 import { InputSelect } from "../../../../data/input_select";
+// tslint:disable-next-line: no-duplicate-imports
+import { PaperInputElement } from "@polymer/paper-input/paper-input";
```

Comment:
We can upgrade to TypeScript 3.8 and make this
```suggestion
import type { PaperInputElement } from "@polymer/paper-input/paper-input";
```

Reply:
Seems like it hasn't made it in Babel yet https://github.com/babel/babel/pull/11053